project/Build.scala

```diff
@@ -169,9 +171,9 @@ object Build {
     if (isRelease)
       baseVersion
     else if (isNightly)
-      baseVersion + "-bin-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
+      baseVersion + "-RC1-bin-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
```

Comment:
```suggestion
      prereleaseVersion + "-bin-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
```

Reply:
idk if this makes more sense

Reply:
@Kordyjan asked me to add this field for a future automation. I'm not sure what it is so I would rather not use it until then. To see if it makes sense to use it in a more general context.